GameData/RP-0/Tree/ProceduralAvionics.cfg

```diff
@@ -27,6 +27,15 @@
 		}
 	}
 	
+	MODULE {
+		name = AdjustableCoMShifter
+		offsetX = 0
+		offsetY = 0
+		offsetZ = 0
+		offsetPercent = 0.2
+		IsDescentMode = false
```

Comment:
No need to define default values inside the patch. Also offsetPercent and IsDescentMode are only used on capsules for the Descent mode functionality.

Reply:
Thanks, I was unaware of this from asking people & reading the documentation. Well, not really any documentation to read. 

So what changes should I make? Remove the offsetPercent and IsDescentMod nodes?

Reply:
Yes, remove all the lines that I marked above.